README.md

```diff
@@ -1,43 +1,62 @@
+
+In your codebase, you can use the javascript-y names, and the provided `airtable.js` file will automatically transform them to and from the actual Airtable column names!
 
 ## Installation
 
-1. Add package as a dev dependency
+#### 1) Add package as a dev dependency
 
 `npm install --save-dev airtable-schema-generator`
 or
 `yarn add -D airtable-schema-generator`
 
-2. Create a `.airtable-schema-generator.env` file.
-
-Create a file called `.airtable-schema-generator.env` (identical to `.env.example`) in the root of your project and fill in the values with your airtable account email and password, as well as your Airtable Base ID. You can find your Base ID in the API URL: airtable.com/`{baseId}`/api/docs.
+#### 2) Choose a mode to use the package in
 
-3. Add new env file to gitignore
+Options: 
+1. **Auto**: You will provide your username and password in a `.airtable.env` file and the schema generator will scrape the airtable API page via an automated web browser
+2. **Auto (Headless)**: Same as above, just without a visible web browser
+3. **Manual**: You manually scrape airtable API page and save to file. No username and password needed
 
-Add the line `.airtable-schema-generator.env` to your `.gitignore` file
-
-4. Configure the input and output folder
+#### 3) Add generator settings to your package.json file
 
 In your `package.json` add the following: 
 ```
 "airtable-schema-generator": { 
-  "input": "path/to/input/folder",
+  "baseId": "<Airtable Base ID>",
+  "mode": "<Desired Mode>",
   "output": "path/to/output/folder"
 }
 ```
-specifying where your `schemaMetadata.json` file lives as the input folder and where you'd like to store your utility functions as the output folder
 
-5. Add convenient run script
+The `output` folder is where you'd like your generated files to live. 
+The mode parameter accepts "auto", "auto-headless", and "manual"
+
+**Other Parameters**: 
+`input`: Only required if mode is set to Manual. Similar to `output`, specifies where `schemaRaw.json` can be found. Details below
+`defaultView`: You can specify a default view to use for Read requests. Default is "Grid View"
+`schemaMeta`: Where your metadata for schema generation lives. Details below
+
+specifying where your `schemaMetadata.json` file lives as the input folder and where you'd like to store your utility functions as the output folder.
+
+#### 3b) If using auto or auto-headless mode, create a `.airtable.env` file.
+
+Create a file called `.airtable.env` (identical to `.env.example`) in the root of your project and fill in the values with your airtable account email and password. This information is only saved on your computer in this hidden file.
```

Comment:
nitpick, but if we're going to call `.airtable.env`, any thoughts on having the example file be called `.example.env` (or vice versa) ?